docs/units-and-global-variables.rst

```diff
@@ -163,6 +163,16 @@ Mathematical and Cryptographic Functions
 ``keccak256(bytes memory) returns (bytes32)``:
     compute the Keccak-256 hash of the input
 
+.. warning::
```

Comment:
Please undo this move, it refers to ecrecover, which is only introduced further down.